ja/lessons/ecto/querying.md

```diff
@@ -90,112 +90,82 @@ iex> Repo.all(query)
 ]
 ```
 
-#### キーワードクエリによる `from` の使用
+#### `from` によるBindinglessなクエリの作成
 
-上の例では `from/2` に _キーワードクエリ_ の引数を与えています。 `from` をキーワードクエリとともに使う場合、第１引数は次の2つのうちいずれかになります:
-
-- `in` 式 (例: `m in Movie`)
-- `Ecto.Queryable` プロトコルを実装したモジュール (例: `Movie`)
-
-第２引数は `select` キーワードクエリになります。
-
-#### クエリ式による `from` の使用
-
-クエリ式とともに `from` を使う場合、第１引数は `Ecto.Queryable` プロトコルを実装した値でなければいけません (例: `Movie`)。第２引数は式となります。例を見てみましょう:
+上の例では、SQLステートメントの最も楽しい部分がありません。特定のフィールドのみを参照したり、何らかの条件でレコードをフィルタリングしたりすることがよくあります。タイトルが `"Ready Player One"` である全ての映画の `title` と `tagline` を取得してみましょう。
 
 ```elixir
-iex> query = select(Movie, [m], m)
-%Ecto.Query<from m in Example.Movie, select: m>
-iex> Repo.all(query)
+iex> query = from(Movie, where: [title: "Ready Player One"], select: [:title, :tagline])
+%Ecto.Query<from m in Example.Movie, where: m.title == "Ready Player One",
+ select: [:title, :tagline]>
 
-06:16:20.854 [debug] QUERY OK source="movies" db=0.9ms
+iex> Repo.all(query)                                                                    
+SELECT m0."title", m0."tagline" FROM "movies" AS m0 WHERE (m0."title" = 'Ready Player One') []
 [
   %Example.Movie{
     __meta__: %Ecto.Schema.Metadata<:loaded, "movies">,
     actors: %Ecto.Association.NotLoaded<association :actors is not loaded>,
     characters: %Ecto.Association.NotLoaded<association :characters is not loaded>,
-    distributor: %Ecto.Association.NotLoaded<association :distributor is not loaded>,
-    id: 1,
+    id: nil,
     tagline: "Something about video games",
     title: "Ready Player One"
   }
 ]
 ```
 
-`in` ステートメント (`m in Movie`) が _必要ない_ 場合にはクエリ式を使うことができます。データ構造への参照を必要としない場合には `in` ステートメントは必要ありません。上のクエリはデータ構造の参照を必要とません。例えば、特定の条件を満たす映画を選択することはない場合がそれにあたります。そのため `in` 式とキーワードクエリを使う必要はありません。
+返ってくる構造体には `tagline` と `title` フィールドのみが設定されていることに注意してください - これは `select:` の部分の結果です。
 
-### `select` 式の使用
+このようなクエリは、bindingを必要としないほど単純であるため、 *bindingless* と呼ばれます。
 
-クエリのselectステートメントの部分を指定するために `Ecto.Query.select/3` 関数を使います。特定のフィールドだけ選択したい場合は、アトムのリスト、もしくは構造体のキーの参照でそれを指定することができます。1つ目のアプローチを見てみましょう:
+#### クエリ式による `from` の使用
```

Comment:
I want to ask why did you translate `Binding in queries` to this. 🤔

Reply:
@riseshia oops! I mistatke translate.
Maybe,  I looked mixed with other parts.

I fix it 👍 
https://github.com/elixirschool/elixirschool/pull/2166/commits/9eefa9afa68eccf2cefb2e388ac8bfea299b8942